content/shmem_malloc.tex

```diff
@@ -34,7 +34,7 @@
     null pointer, no action occurs. 
            
     The \FUNC{shmem\_realloc} routine changes the size of the block to which
-    \VAR{ptr} points to the size (in bytes) specified by \VAR{size}.  The contents
+    \VAR{ptr} points to the size, in bytes, specified by \VAR{size}.  The contents
```

Comment:
I find this parentheses-to-comma change to be distracting as a reader. It may not be the stylistic preference of @j-mike-culhane, however, the specification makes regular (if not consistent) use of parentheses throughout the text.